app/controllers/shipit/api/deploys_controller.rb

```diff
@@ -13,12 +13,14 @@ def index
         accepts :force, Boolean, default: false
         accepts :require_ci, Boolean, default: false
         accepts :env, Hash, default: {}
+        accepts :safeties_enforced, Boolean, default: false
```

Comment:
Should this default to `true`?

Reply:
The purpose of this param is a hard override of `ignore_safeties` (as in, we already enforced safeties before this API request) - so it's defaulting to `false` since we want safeties to usually be reported on. Naming this is tricky so I'm open to suggestions!

Reply:
ah, this is purely to switch the DB field so that it's displayed in the UI? 

Then I don't think adding a parameter is the clean way to do this. It would be much better to automatically set that to true if the `force` was actually needed.